src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -215,6 +215,11 @@ public abstract class Opcode
             "+-------------------------------------------------------------------------+\n";
 
         public int Id { get { return id; } }
+        /// <summary>
+        /// How far to jump to the next opcode.  1 is typical (just go to the next opcode),
+        /// but in the case of jump and branch statements, it can be other numbers.  This will
+        /// be ignored if the CPU has been put into a yield state with CPU.YieldProgram().
+        /// </summary>
         public int DeltaInstructionPointer { get; protected set; }
```

Comment:
After I posted it I looked back and realized that changing the pointer is still very possible.  Since the IP changes when the cpu begins yielding, a function or opcode is free to change the instruction pointer again just before returning true in `YieldFinishedDetector.IsFinished`